src/app/recipes/components/recipe-details/recipe-details.component.html

```diff
@@ -0,0 +1,74 @@
+<div class="container">
+    <div class="details">
+        <div class="details-name">
+            <i class="material-icons" (click)="toggleBookmarkRecipe()">{{ isRecipeBookmarked() ? 'bookmark' : 'bookmark_border' }}</i>
```

Comment:
Direct calls to methods should not be used in templates. Since here you don't have any params, would calculate that flag just once in component.